Apps/phdigitalshadows/digital_shadows.json

```diff
@@ -1207,52 +1355,44 @@
                     ]
                 }
             ],
+            "render": {
+                "type": "table",
+                "width": 1,
+                "height": 5,
+                "title": "Digital Shadows Get Data Breach Record"
+            },
             "versions": "EQ(*)"
         },
         {
             "action": "get breachrecord byuser",
             "identifier": "get_data_breach_record_by_username",
-            "description": "This command allows you to search breach records based on the domain, review status, or full or partial strings from username",
+            "description": "This action allows you to search breach records based on the domain, review status, or full/partial strings from the username",
             "type": "investigate",
             "read_only": true,
             "parameters": {
                 "user_name": {
+                    "description": "User Name (use '*' for wildcard matching)",
                     "data_type": "string",
-                    "order": 0,
-                    "description": "User name (use '*' for wildcard matching)",
-                    "required": true
+                    "required": true,
+                    "order": 0
                 },
                 "domain_names": {
+                    "description": "Domain Names (comma separated)",
                     "data_type": "string",
-                    "order": 1,
-                    "description": "Domain names (comma separated)"
+                    "order": 1
                 },
                 "published_date_range": {
+                    "description": "Published Date Range (ex:- P30M)",
```

Comment:
Explain notation. P30M is unclear to me

Reply:
This is outdated code which is no longer in the source code.

Reply:
Hey @somya-metron, 
Actually, we have moved the example values to the output datapath for this parameter as well as in other similar cases, as this is the practice we follow, so if you can explain this notation here then I can replace it in the code.  Also at some places,  P30D has been used as an example value.

Reply:
Thanks @amangal-crest The example value P30D stands for "Past 30 Days". It is used to provide the number of past days for which data needs to be polled. About the "password" field in "search databreach records" action, let me get back to you after discussing with our internal dev team.

Reply:
Fixed the "password" parameter, thanks!

Reply:
Hi @somya-metron
This is regarding the "password" parameter of "search databreach records" action, I was testing this action for negative scenarios and found some different results than earlier runs, I believe this behavior is due to the change in datatype of the "password" parameter.
Have you tested the action after changing the datatype? if not then please do so.

Reply:
Hi @amangal-crest thanks for the update.
We would suggest to keep the datatype of the field as "string" as it would be used by user to find data breach records containing that particular string. We will be testing the same at our end as well.

Reply:
@somya-metron
Okay then, Please change its datatype to "string" or let me know if I have to do it, so afterward we can release this app.

Reply:
I have made the change in the latest commit.